include/translated-term.php

```diff
@@ -318,4 +313,29 @@ public function set_language_in_mass( $ids, $lang ) {
 			$this->set_translation_in_mass( $translations );
 		}
 	}
+
+	/**
+	 * Returns database-related informations that can be used in some of this class methods.
+	 * These are specific to the table containing the objects.
+	 *
+	 * @since 3.4.3
+	 * @see PLL_Translatable_Object::join_clause()
+	 * @see PLL_Translatable_Object::get_objects_with_no_lang_sql()
```

Comment:
```suggestion
	 * @see PLL_Translatable_Object::join_clause()
	 * @see PLL_Translatable_Object::get_objects_with_no_lang_sql()
	 
	 * @since 3.4.3
```